code/modules/modular_computers/terminal/commands/file.dm

```diff
@@ -4,7 +4,6 @@
 	man_entry = list(
 		"Format: file \[-flag filename \[filename2\]\]",
 		"Without options, list all files on storage devices.",
-		"With -h followed by filename, toggle the hidden flag on the file.",
```

Comment:
Just wondering why was this one removed?

Reply:
hidden files do not exist in our scuffed half broken modular computer code and i cba to port them